src/api/app/models/kiwi/image/to_xml.rb

```diff
@@ -1,7 +1,6 @@
-# TODO: Please overwrite this comment with something explaining the model target
 module Kiwi
   class Image
-    class Xml
+    class ToXml
```

Comment:
We already talked about not having ToXml as a name for a class... any other options?

Reply:
> any other options?

Maybe you can suggest some other options? Since you are the one who doesn't like the proposed naming :P

Reply:
> Maybe you can suggest some other options? Since you are the one who doesn't like the proposed naming :P

Done in the other comment.. 👍